server.js

```diff
@@ -1,29 +1,64 @@
 import express from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
+const listEndpoints = require("express-list-endpoints")
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+ 
+const documentation = {
```

Comment:
I used the listEndPoints too, but not quite like this! Interesting to see that different solutions work!
Mine was: `res.send(listEndPoints(app))`. Yours is sent with json format, that seems cool!